bitcoin-explained/breaking-down-taproot-activation-options.md

```diff
@@ -0,0 +1,1674 @@
+---
+title: "Breaking Down Taproot Activation Options"
+transcript_by: Sjors via rev.com
+revision_by: edilmedeiros

```

Comment:
We don't currently support this `revision_by` field so it will never show up on the website. I haven't put any thought into how we should handle/display revisions. Maybe you can include it inside the `transcript_by` field?
```suggestion
transcript_by: Sjors, edilmedeiros
```

Reply:
All right. Maybe it's a good ide to open an issue about revisions and think about how to handle them.